tremor-cli/src/test.rs

```diff
@@ -178,7 +181,37 @@ async fn run_integration(
         status::tags(&tags, Some(&matched), Some(&config.excludes))?;
 
         // Run integration tests
-        let test_report = process::run_process("integration", base, root, &tags).await?;
+        let test_report = if let Some(timeout) = config.timeout {
+            match process::run_process("integration", base, root, &tags)
+                .timeout(timeout)
+                .await
+            {
+                Err(_) => {
+                    // timeout
+                    // walk the entire dir and print all *.log files for debuggability
+                    error!("Timeout running integration test {}", root.display());
+                    let walker = GlobWalkerBuilder::new(root, "*.log")
```

Comment:
I don't think we need this, if the test fails we upload the log files in CI, removing the printer could solve the coverage?

Reply:
yeah, makes sense. i didn't see that we store the log files. It should be enough to error out here. Good point!

Reply:
It's well hidden, and I think we enver used it so it's a secret :) shhhh don't tell anyone!